appserver/admin/cli-optional/src/main/java/com/sun/enterprise/admin/cli/optional/DerbyControl.java

```diff
@@ -125,10 +126,10 @@ public DerbyControl(final String derbyCommand, final String derbyHost, final Str
 
         // Do not set derby.system.home if derbyHome is empty
         if (!isEmpty(derbyHome)) {
-            System.setProperty("derby.system.home", derbyHome);
+            setProperty("derby.system.home", derbyHome, true);
```

Comment:
With all usages updated, could we have `setProperty(String, String)`, that would default the 3rd parameter to `true`?